src/chapters/more-on-functions/recursion-walkthrough2.rst

```diff
@@ -0,0 +1,133 @@
+   :alt: Visual representation for the third-easiest case.
+
+As before, ``combineEntries`` executes the ``else`` statement, and we can
+follow the logic:
+
+a. ``combineEntries`` returns ``'L'`` and calls itself again using what is left
+   inside the array (``['C', '1']``).
+b. When passed ``['C', '1']``, ``combineEntries`` returns ``'C'`` and calls
+   itself again using what is left inside the array (``['1']``).
+c. When passed ``['1']``, which is the base case, ``combineEntries`` returns
+   ``'1'``.
+d. The strings ``'C'`` and ``'1'`` get combined and returned.
+e. The strings ``'L'`` and ``'C1'`` get combined and returned as the final
+   result.
+
+As we make the array longer, ``combineEntries`` calls itself more times. Each
+call evaluates a smaller and smaller section of the array until reaching the
+base case. This sets up a series of return events - each one selecting a
+single entry from the array. Rather than building ``'LC101'`` from left to
+right, recursion constructs the string starting with the base case and
+adding new characters to the front:
+
+| ``'1'``
+| ``'01'``
+| ``'101'``
+| ``'C101'``
+| ``'LC101'``
+
+A Function Calls Itself
+------------------------
+
+So how do we code the ``else`` statement in ``combineEntries``? Recall what
+needs to happen each time the statement runs:
+
+a. Select the first element in the array,
+b. Call ``combineEntries`` again with a smaller array.
+
+Bracket notation takes care of part a: ``arrayName[0]``.
+
+| For part b, remember that the :ref:`slice method <slice-examples>` returns
+   selected entries from an array. To return everything BUT the first entry in
+| ``arr = ['L', 'C', '1', '0', '1']``, use ``arr.slice(1)``.
+
+Let's add the bracket notation and the ``slice`` method to our function:
+
+.. sourcecode:: js
+
+   function combineEntries(arrayName){
+      if (arrayName.length === 1){
+         return arrayName[0];
+      } else {
+         return arrayName[0]+combineEntries(arrayName.slice(1));
+      }
+   }
+
+Each time the ``else`` statement runs, it extracts the first element in the
+array with ``arrayName[0]``, then it calls itself with the remaining array
+elements (``arrayName.slice(1)``). For ``combineElements(['L', 'C', '1', '0', '1']);``:
+
+| a. First call: Combine ``'L'`` with ``combineEntries(['C', '1', '0', '1'])``.
+| b. Second call: Combine ``'C'``, with ``combineEntries(['1', '0', '1'])``.
+| c. Third call: Combine ``'1'``, with ``combineEntries(['0', '1'])``.
+| d. Fourth call: Combine ``'0'``, with ``combineEntries(['1'])``.
+| e. Fifth call: Base case returns ``'1'``.
+
+| To get the final result, proceed *up the chain*:
+| e. Return ``'1'`` to the fourth call,
+| d. Return ``'01'`` to the third call,
+| c. Return ``'101'`` to the second call,
+| b. Return ``'C101'`` to the first call,
+| a. Return ``'LC101`` as the final result.
+
+See recursion in action `here <https://repl.it/@launchcode/RecursionExample01>`__.
```

Comment:
Make this a [semantic link](https://education.launchcode.org/curriculum-docs/guide/style.html#linking).

Reply:
Fixed.